Tests/Schema.NET.Test/Examples/BookTest.cs

```diff
@@ -1,218 +1,213 @@
+                        "}," +
+                        "\"expectsAcceptanceOf\":{" +
+                            "\"@type\":\"Offer\"," +
+                            "\"availability\":\"https://schema.org/InStock\"," +
+                            "\"eligibleRegion\":{" +
+                                "\"@type\":\"Country\"," +
+                                "\"name\":\"US\"" +
+                            "}," +
+                            "\"price\":6.99," +
+                            "\"priceCurrency\":\"USD\"" +
+                        "}" +
+                    "}," +
+                    "\"bookEdition\":\"2nd Edition\"," +
+                    "\"bookFormat\":\"https://schema.org/Hardcover\"," +
+                    "\"isbn\":\"031676948\"" +
+                "}," +
+                "{" +
+                    "\"@type\":\"Book\"," +
+                    "\"potentialAction\":{" +
+                        "\"@type\":\"ReadAction\"," +
+                        "\"target\":{" +
+                            "\"@type\":\"EntryPoint\"," +
+                            "\"actionPlatform\":[" +
+                                "\"https://schema.org/DesktopWebPlatform\"," +
+                                "\"https://schema.org/IOSPlatform\"," +
+                                "\"https://schema.org/AndroidPlatform\"" +
+                            "]," +
+                            "\"urlTemplate\":\"https://www.barnesandnoble.com/store/info/offer/031676947?purchase=true\"" +
+                        "}," +
+                        "\"expectsAcceptanceOf\":{" +
+                            "\"@type\":\"Offer\"," +
+                            "\"availability\":\"https://schema.org/InStock\"," +
+                            "\"eligibleRegion\":{" +
+                                "\"@type\":\"Country\"," +
+                                "\"name\":\"UK\"" +
+                            "}," +
+                            "\"price\":1.99," +
+                            "\"priceCurrency\":\"USD\"" +
+                        "}" +
+                    "}," +
+                    "\"bookEdition\":\"1st Edition\"," +
+                    "\"bookFormat\":\"https://schema.org/EBook\"," +
+                    "\"isbn\":\"031676947\"" +
+                "}" +
+            "]" +
+        "}";
+
+        [Fact]
+        public void Deserializing_BookJsonLd_ReturnsMatchingBook()
+        {
+            Assert.Equal(this.book.ToString(), SchemaSerializer.DeserializeObject<Book>(this.json)!.ToString());
+            Assert.Equal(SchemaSerializer.SerializeObject(this.book), SchemaSerializer.SerializeObject(SchemaSerializer.DeserializeObject<Book>(this.json)!));
+        }
+
+        [Fact]
+        public void Deserializing_BookJsonLd_ReturnsBook()
+        {
+            var book = SchemaSerializer.DeserializeObject<Book>(this.json)!;
+
+            Assert.Equal("The Catcher in the Rye", book.Name);
```

Comment:
Here is an example of where we could use Verify. Wherever we have long lists of Asserts.